infra/helper.py

```diff
@@ -205,6 +224,8 @@ def build_fuzzers(build_args):
 def run_fuzzer(run_args):
   """Runs a fuzzer in the container."""
   parser = argparse.ArgumentParser('helper.py run_fuzzer')
+  _add_engine_args(parser)
```

Comment:
yes I was planning to add this in a future CL. this change is mostly to get the ball rolling with builds (things like reproduce doesn't work either).